### PR TITLE
Add contributing guidelines file

### PR DESCRIPTION
**Pull request by edakturk14**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+This guide aims to provide overview of the contribution workflow to help us make the contribution process effective for everyone involved. 
+
+## Table of Contents
+1. [About the Project](#about-the-project)
+1. [Project Status](#project-status)
+1. [Getting Started](#getting-started)
+
+## About the Project
+
+Scaffold-Eth 2 is a starter toolkit to build applications on Ethereum. It's a minimal and forkable repo providing builders a MVP to create decentralized applications.
```
Comment (carletex) [inline CONTRIBUTING.md:14]: Maybe we can find an alternative for MVP? Something like "starter kit"

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
+# Welcome to Scaffold-ETH 2 Contributing Guide 
+
+Thank you for investing your time in contributing to Scaffold-Eth 2!  
+
+This guide aims to provide overview of the contribution workflow to help us make the contribution process effective for everyone involved. 
+
+## Table of Contents
+1. [About the Project](#about-the-project)
+1. [Project Status](#project-status)
+1. [Getting Started](#getting-started)
```
Comment (carletex) [inline CONTRIBUTING.md:10]: The numbers are off here and also we are not using numbers on the Sections.

We could either update the numbers here (and add them to the Sections titles) or remove them completely.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+- Solve open issues 
+- Report bugs or feature requests
+- Improve the documentation 
+
+Contributions are made to via Issues and Pull Requests (PRs). A few general guidelines for contributions:
+
+- Search for existing Issues and PRs before creating your own.
+- Contributions should only fix/add the functionality in the issue OR address style issues, not both.
+- If you’re running into an error please give context and explain what you’re trying to do and how to reproduce the error. 
+- Please use the same formatting in the code repository.
```
Comment (carletex) [inline CONTRIBUTING.md:34]: We can add here something like: "You can configure you IDE to do it for you by using the prettier / linting config files included on each package"

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+- Report bugs or feature requests
+- Improve the documentation 
+
+Contributions are made to via Issues and Pull Requests (PRs). A few general guidelines for contributions:
+
+- Search for existing Issues and PRs before creating your own.
+- Contributions should only fix/add the functionality in the issue OR address style issues, not both.
+- If you’re running into an error please give context and explain what you’re trying to do and how to reproduce the error. 
+- Please use the same formatting in the code repository. 
+- Include the test files if you are modifying the code.
```
Comment (carletex) [inline CONTRIBUTING.md:35]: We don't have tests for now :D

We should definitely have on the test contract, but not a priority on the front-end IMO

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+
+### Issues
+
+Issues should be used to report problems, request a new feature, or to discuss potential changes before a PR is created.
+
+#### Solve an issue
+
+Scan through our [existing issues](https://github.com/scaffold-eth/se-2/issues) to find one that interests you. 
+
+If there is a contributor working on the issue they will be assigned to the individual. If you find an issue to work on, you are welcome to open a PR with a fix for it.
```
Comment (carletex) [inline CONTRIBUTING.md:46]: maybe "you are welcome to assign it to you and open a PR...."

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -0,0 +1,68 @@
...
+
+#### Create a new issue
+
+If a related issue doesn't exist, you can open a new issue.
```
Comment (carletex) [inline CONTRIBUTING.md:50]: I'll add some tips here like:

- Over-communicate / As much context as possible
- Steps to reproduce / Reason to add the feature.
- Screenshots
- Videos, etc.
- Context

File: CONTRIBUTING.md [hunk 7]
```diff
@@ -0,0 +1,68 @@
...
+
+1. Fork the repo 
+2. Clone the project
+3. Create a new branch with a descriptive name
+4. Commit your changes to the new branch
+5. Push changes to your fork
+6. Open a PR in our repository and tag one of the maintainers to review your PR. 
+
+Once you submit your PR:
+-  We may ask questions or request for additional information.
```
Comment (carletex) [inline CONTRIBUTING.md:66]: Extra space here :D

File: CONTRIBUTING.md [hunk 8]
```diff
@@ -0,0 +1,68 @@
...
+- We may ask for changes to be made before a PR can be merged.
+- As you update your PR and apply changes, mark each conversation as resolved.
```
Comment (carletex) [inline CONTRIBUTING.md:68]: At the end, I'll add something like: 

"Once the PR is approved, we'll "squash-and-merge" to keep the git commit history clean."

File: CONTRIBUTING.md [hunk 9]
```diff
@@ -0,0 +1,68 @@
...
+#### Pull Request Process
+
+We follow the ["fork-and-pull" Git workflow](https://github.com/susam/gitpr)
+
+1. Fork the repo 
+2. Clone the project
+3. Create a new branch with a descriptive name
+4. Commit your changes to the new branch
+5. Push changes to your fork
+6. Open a PR in our repository and tag one of the maintainers to review your PR.
```
Comment (carletex) [inline CONTRIBUTING.md:63]: Maybe add some tips here also? Like in ISSUES

File: CONTRIBUTING.md [hunk 10]
```diff
@@ -0,0 +1,68 @@
+# Welcome to Scaffold-ETH 2 Contributing Guide 
+
+Thank you for investing your time in contributing to Scaffold-Eth 2!  
+
+This guide aims to provide overview of the contribution workflow to help us make the contribution process effective for everyone involved. 
+
+## Table of Contents
```
Comment (carletex) [inline CONTRIBUTING.md:7]: I'd love to have a "Vision" or "Philosophy" or something like that to explain about the simple & minimal philosophy, iterations, adding is easier than removing.... and we want to put a lot of intention on it.

**Review by carletex — CHANGES_REQUESTED** [review]
Thanks for this @edakturk14 !

This is going to be super helpful to align. I've just added some review comments. Happy to keep iterating on this ;)

File: CONTRIBUTING.md (hunk 10 above)
Comment (edakturk14) [reply CONTRIBUTING.md]: Added in the about-project section.

**Comment by edakturk14** [comment]
Hey @carletex! thanks for the feedback, super helpful. I have made the changes, please lmk if any change is needed.

**Review by carletex — APPROVED** [review]

**Pull request merged**